store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/impl/FederatedAddGraphHandlerTest.java

```diff
@@ -98,7 +91,7 @@ public void shouldAddGraph() throws Exception {
         store.initialise(FEDERATEDSTORE_GRAPH_ID, null, federatedStoreProperties);
         final Schema expectedSchema = new Schema.Builder().build();
 
-        assertThat(store.getGraphs(testUser, null, ignore)).hasSize(0);
+        assertThat(store.getGraphs(testUser, null, new AddGraph())).hasSize(0);
```

Comment:
Definitely missing some context, but why did this change from `IgnoreOptions` to `AddGraph`?

Reply:
IgnoreOptions was a TestClass created I think which did just that.
Now options is storing Federation flags its required.